docs/docs/getting_started/tooling/noir_codegen.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Noir Codegen for Typescript
+description: Learn how to use Noir codegen to generate Typescript bindings 
+keywords: [Nargo, Noir, compile, TypeScript]
+sidebar_position: 2
+---
+
+You can generate TypeScript bindings with `noir_codegen`, allowing for easier and faster integration of your Noir circuits into TypeScript projects.
+
+It is used in conjunction with `nargo export` which generates JSON for specified functions.
```

Comment:
Is this a new command? I don't see it documented on the commands page, or in the CLI help

![image](https://github.com/noir-lang/noir/assets/18372439/9a232daf-bffc-4167-b696-96becefddda8)

Reply:
The "specified functions" part is correct, but doesn't say how (here or later). Will add below.